opennms-provision/opennms-detector-simple/blueprint-detector.xml

```diff
@@ -0,0 +1,50 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+	xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0"
+	xsi:schemaLocation="
+		http://www.osgi.org/xmlns/blueprint/v1.0.0 
+		http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+		http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0
+		http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
+">
+
+	<bean id="icmp" class="org.opennms.netmgt.provision.detector.icmp.IcmpDetector">
+	</bean>
+	<service id="icmpDetector" ref="icmp"
+		interface="org.opennms.netmgt.provision.SyncServiceDetector">
+		<service-properties>
+			<entry key="alias" value="detector.icmp" />
```

Comment:
Can we change these aliases to be:
```
<entry key="implementation" value="org.opennms.netmgt.provision.detector.icmp.IcmpDetector" />
```
instead of using another constant for the value?